### PR TITLE
Add reaction when someone sends "this is so sad"

### PR DESCRIPTION
**Pull request by Davvos11**
"Alexa, play despacito"

### CODE REVIEW

File: cogs/fun.py [hunk 1]
```diff
@@ -30,6 +30,17 @@ def __init__(self, bot: commands.Bot):
...
+    # Replies "Alexa, play Despacito" to messages containing "this is so sad"
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message):
+        if not self.enabled:
+            return
+        if message.author.bot:
+            return
+        if not "this is so sad" in message.content.lower():
+            return
+        channel = message.channel
```
Comment (Yuhanun) [inline cogs/fun.py]: Omit variable assignment

File: cogs/fun.py [hunk 2]
```diff
@@ -30,6 +30,17 @@ def __init__(self, bot: commands.Bot):
...
+        await channel.send("Alexa, play Despacito")
```
Comment (Yuhanun) [inline cogs/fun.py]: instead do `await message.channel.send(...)`

**Review by Yuhanun — CHANGES_REQUESTED** [review]
Nice pep8 compliance

File: cogs/fun.py (hunk 1 above)
Comment (Davvos11) [reply cogs/fun.py]: done

**Pull request merged**